spec/features/cities_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+feature 'City' do
+
+  scenario 'adds a new city' do
+    region = create(:region)
+    expect{
+      create_region(region)
+      create_city
```

Comment:
you're creating a region and city independently, which means the city is not associated to that region.

A good way to prevent this is add NOT NULL restriction to `cities#region_id` column, also `creaty_city` helper macro could receive the region as argument, so both get linked